language/move-prover/tests/sources/functional/arithm.exp

```diff
@@ -9,7 +9,7 @@ error: abort not covered by any of the `aborts_if` clauses
      │ ╰─────^
      ·
  126 │         x / y
-     │           - abort happened here with execution failure
+     │           - abort happened here
```

Comment:
Flagging this to make sure the change in error message is intended.

Reply:
Yep, unfortunately the code is more often not available than before, but it seems acceptable because in other cases it is.